setup.py

```diff
@@ -82,10 +82,11 @@ def create_readme_rst():
           "torch >= 0.3.0",
           "unidecode",
           "inflect",
-          "librosa",
+          "librosa == 0.5.1",
```

Comment:
Could this be loosened? I'm using developement version of librosa.

Reply:
For me, using stable latest version(from pip) caused synthesis error. (https://github.com/librosa/librosa/issues/640)
Did it get fixed (it is for dev. version but for stable release)?

Reply:
I don't use `librosa.output.write_wav` anymore so https://github.com/librosa/librosa/issues/640 won't be a problem for me (and the repo). I can fix issues if you give me reproducible code. If you have code calling `librosa.output.write_wav` locally, try replacing it with [scipy.io.wavefile.write](https://docs.scipy.org/doc/scipy-0.19.1/reference/generated/scipy.io.wavfile.write.html).

Reply:
Discovered it was due to my own modifications. Thanks :)